tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -15341,6 +15357,7 @@ void DiagnoseEntry(Sema &S, FunctionDecl *FD) {
   case DXIL::ShaderKind::Mesh:
   case DXIL::ShaderKind::Amplification:
   case DXIL::ShaderKind::Invalid:
+  case DXIL::ShaderKind::Compute:
```

Comment:
I think this change will conflict with a change that has already addressed this in the target branch, so you could revert it.